python/tk_desktop/project_menu.py

```diff
@@ -68,6 +68,10 @@ def clear_actions(self):
 
         # Loop through actions and delete the ones listed before the separator
         for action in actions:
+            # Do not delete the Jump To Filesystem menu. This one is hidden on demand instead.
+            if action == self._parent.ui.actionProject_Filesystem_Folder:
```

Comment:
i realise this is a small fix, but wondering if there is something more toolkit-onic we can do to solve it? So that the app decides whether the action should be shown, not the engine. The engine shouldn't know about the apps - if i am understanding this correctly, we are hard coding an app into the engine code, right? Could we use the event system that nuke studio uses to signal? Could we be smart about registering commands so that the app doesn't register the menu entry (command) if the path cache isn't there?

Reply:
JF explained that this is all in the engine - which confused me! That means it's all good to go!